thirdparty/jquery-entwine/dist/jquery.entwine-dist.js

```diff
@@ -748,35 +748,6 @@ Sizzle is good for finding elements for a selector, but not so good for telling
 /* src/jquery.focusinout.js */
 
 (function($){	
-	
-	/**
-	 * Add focusin and focusout support to bind and live for browers other than IE. Designed to be usable in a delegated fashion (like $.live)
-	 * Copyright (c) 2007 Jörn Zaefferer
-	 */
-	if ($.support.focusinBubbles === undefined)  {
-		$.support.focusinBubbles = !!($.browser.msie);
-	}
-
-	if (!$.support.focusinBubbles && !$.event.special.focusin) {
-		// Emulate focusin and focusout by binding focus and blur in capturing mode
-		$.each({focus: 'focusin', blur: 'focusout'}, function(original, fix){
-			$.event.special[fix] = {
-				setup: function(){
-					if (!this.addEventListener) return false;
-					this.addEventListener(original, $.event.special[fix].handler, true);
-				},
-				teardown: function(){
-					if (!this.removeEventListener) return false;
-					this.removeEventListener(original, $.event.special[fix].handler, true);
-				},
-				handler: function(e){
-					arguments[0] = $.event.fix(e);
-					arguments[0].type = fix;
-					return $.event.handle.apply(this, arguments);
-				}
-			};
-		});
-	}
```

Comment:
`$.support` is deprecated and `$.browser` doesn't exist - and anyway we aren't interested in supporting such old browsers so the easy fix to the errors this caused was to remove it.